iothub/device/src/Transport/AmqpIoT/AmqpIoTMessageConverter.cs

```diff
@@ -339,7 +339,7 @@ private static bool TryGetNetObjectFromAmqpObject(object amqpObject, MappingType
             netObject = null;
             if (amqpObject == null)
             {
-                return false;
```

Comment:
Is this `amqpObject` specific to application properties for a message only? Is it being passed from any other method? If yes, then we are introducing new behavior where any `amqpObject` can now be `null`.

Reply:
Nope. I see only one reference for messages